### ISSUE TITLE
stuck on erigon version 2022.07.2-alpha-c7a94eee with no error messages

### ISSUE THREAD

**Issue by zhonglinwei** [ledgerwatch/erigon#4718]
Logs:
[INFO] [07-15|03:30:32.562] RPC Daemon notified of new headers       from=15144851 to=15144852 header sending=9.463µs log sending=217ns
[INFO] [07-15|03:30:32.562] [1/16 Headers] Waiting for headers...    from=15144852
[INFO] [07-15|03:30:35.933] [snapshots] Dumping txs                  block num=14577933 alloc=4.5GB sys=5.8GB
[INFO] [07-15|03:30:39.492] [1/16 Headers] Processed                 highest inserted=15144853 age=10s
[INFO] [07-15|03:30:39.614] [6/16 Execution] Completed on            block=15144853
[INFO] [07-15|03:30:40.151] Timings (slower than 50ms)               Headers=6.93s Execution=106ms HashState=86ms IntermediateHashes=147ms Prune TxLookup=83ms
[INFO] [07-15|03:30:40.151] Tables                                   PlainState=69.6GB AccountChangeSet=1.5GB StorageChangeSet=2.6GB BlockTransaction=68.1GB TransactionLog=6.7GB FreeList=436.4MB ReclaimableSpace=436.4GB
[INFO] [07-15|03:30:40.341] Commit cycle                             in=189.780458ms
[INFO] [07-15|03:30:40.342] RPC Daemon notified of new headers       from=15144852 to=15144853 header sending=9.247µs log sending=272ns
[INFO] [07-15|03:30:40.342] [1/16 Headers] Waiting for headers...    from=15144853
[INFO] [07-15|03:30:55.962] [snapshots] Dumping txs                  block num=14578455 alloc=2.7GB sys=5.8GB
[INFO] [07-15|03:31:00.343] [1/16 Headers] Wrote block headers       number=15144853 blk/second=0.000 alloc=2.9GB sys=5.8GB
[INFO] [07-15|03:31:12.851] [1/16 Headers] Processed                 highest inserted=15144854 age=33s
[INFO] [07-15|03:31:13.819] [6/16 Execution] Completed on            block=15144854
[INFO] [07-15|03:31:15.950] [snapshots] Dumping txs                  block num=14578954 alloc=3.4GB sys=5.8GB
[INFO] [07-15|03:31:16.280] Timings (slower than 50ms)               Headers=32.509s Bodies=195ms Execution=740ms HashState=546ms IntermediateHashes=828ms CallTraces=132ms AccountHistoryIndex=80ms StorageHistoryIndex=180ms LogIndex=92ms TxLookup=103ms Prune TxLookup=242ms Prune LogIndex=125ms Prune AccountHistoryIndex=89ms
[INFO] [07-15|03:31:16.280] Tables                                   PlainState=69.6GB AccountChangeSet=1.5GB StorageChangeSet=2.6GB BlockTransaction=68.1GB TransactionLog=6.7GB FreeList=436.4MB ReclaimableSpace=436.4GB
[INFO] [07-15|03:31:17.234] Commit cycle                             in=954.496814ms
[INFO] [07-15|03:31:35.945] [snapshots] Dumping txs                  block num=14579494 alloc=3.9GB sys=5.8GB

**Comment by AskAlexSharov** [comment]
In your logs i see that you are fine. Where do you see stuck?

**Comment by zhonglinwei** [comment]
> In your logs i see that you are fine. Where do you see stuck?

The log doesn't scroll anymore, after 90 minutes, I check log again, it still here.

**Comment by zhonglinwei** [comment]
> > In your logs i see that you are fine. Where do you see stuck?
> 
> The log doesn't scroll anymore, after 90 minutes, I check log again, it still here.

<img width="881" alt="image" src="https://user-images.githubusercontent.com/51225774/179154504-66e13140-5353-45fd-a39e-78e9bc8ecfff.png">

**Comment by AskAlexSharov** [comment]
Wow. Can you try: 
pkill -SIGUSR1 erigon

**Comment by zhonglinwei** [comment]
> 

yes, i try to restart erigon docker, it will sync block again, but it will stuck again in about 25 minutes. 
<img width="916" alt="image" src="https://user-images.githubusercontent.com/51225774/179190201-c6ee5230-b327-46a6-8344-6ac937a8d18b.png">

**Comment by AskAlexSharov** [comment]
@du5 Can you try:
pkill -SIGUSR1 erigon

**Comment by mandrigin** [comment]
@du5 there is no output from pkill in this screenshot, it prints goroutines.
Though, some good news is that if you try the latest develop version the bug might go away. Try and let me know how it goes.

**Comment by mandrigin** [comment]
Try to build the latest version from our `devel` branch and see if it repeats there.

**Comment by du5** [comment]
#4798

**Comment by cifer76** [comment]
I just met the similar issue, my rpcdaemon and erigon core are running in different pods, when the erigon core stuck I caught a pprof for it and it shows during the stuck most of the cpu time is on gc:

![image](https://user-images.githubusercontent.com/4777457/184293955-2e584964-9016-4c60-b07f-e4147c2c6216.png)

**Comment by revitteth** [comment]
I'll close this - let's reopen against a newer version if the issue still persists. :)
